FavoriteArtists/FavoriteArtists/FavoriteArtistsTableViewController.m

```diff
@@ -0,0 +1,74 @@
+//
+//  FavoriteArtistsTableViewController.m
+//  FavoriteArtists
+//
+//  Created by Clayton Watkins on 9/25/20.
+//
+
+#import "FavoriteArtistsTableViewController.h"
+#import "Artist.h"
+#import "SearchDetailViewController.h"
+#import "ArtistPersistenceController.h"
+
+@interface FavoriteArtistsTableViewController ()
+
+@property ArtistPersistenceController *persistenceController;
+
+@end
+
+@implementation FavoriteArtistsTableViewController
+
+#pragma mark - LifeCycle
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.persistenceController = [[ArtistPersistenceController alloc] init];
+}
+
+- (void)viewWillAppear:(BOOL)animated
+{
+    [super viewWillAppear:animated];
+    
+    [self.tableView reloadData];
+}
+
+#pragma mark - Table view data source
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    return self.persistenceController.artists.count;
+}
+
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"ArtistCell" forIndexPath:indexPath];
+    
+    Artist *artist = [self.persistenceController.artists objectAtIndex:indexPath.row];
```

Comment:
Consider implementing artistAtIndex as a faster accessor to get the specific artist from the internal array directly.